src/charm.py

```diff
@@ -366,6 +366,17 @@ def _create_manifests(self, manifest_files, context):
             manifests.append(manifest)
         return json.dumps(manifests)
 
+    def _send_ingress_info(self, interfaces):
+        if interfaces["ingress"]:
+            interfaces["ingress"].send_data(
+                {
+                    "prefix": "/mlflow/",
+                    "rewrite": "/",
+                    "service": self.model.app.name,
+                    "port": int(self._port),
```

Comment:
Let's add the namespace here too.  If we omit it, then it assumes you're in the same namespace as istio-pilot which might not be true.  
```suggestion
                    "prefix": "/mlflow/",
                    "rewrite": "/",
                    "service": self.model.app.name,
                    "namespace": self.model.name,
                    "port": int(self._port),
```